<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="line_x">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Agregar inicio</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_lista">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>251</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_blue">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>220</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_green">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_salir_lista">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Regresar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_voltaje">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_g">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>G:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addf">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Agregar final</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_id">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_id">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Id:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_red">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_r">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>R:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_lista">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lista</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_salir">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_b">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>B:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_y">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Posicion Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_voltaje">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Voltaje:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Posicion X:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_y">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_tabla">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tabla">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tabla</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_buscar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buscar Id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_salir_tabla">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Regresar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_buscar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mostrar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mostrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_salir_mostrar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Regresar</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViews_ellipses">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>291</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
   </widget>
   <zorder>graphicsViews_ellipses</zorder>
   <zorder>pushButton_salir_mostrar</zorder>
   <zorder>line_buscar</zorder>
   <zorder>pushButton_salir_tabla</zorder>
   <zorder>pushButton_buscar</zorder>
   <zorder>tableWidget_tabla</zorder>
   <zorder>plainTextEdit_lista</zorder>
   <zorder>line_blue</zorder>
   <zorder>line_green</zorder>
   <zorder>pushButton_salir_lista</zorder>
   <zorder>line_voltaje</zorder>
   <zorder>label_g</zorder>
   <zorder>pushButton_addf</zorder>
   <zorder>line_id</zorder>
   <zorder>label_id</zorder>
   <zorder>line_red</zorder>
   <zorder>label_r</zorder>
   <zorder>pushButton_lista</zorder>
   <zorder>pushButton_salir</zorder>
   <zorder>label_b</zorder>
   <zorder>label_y</zorder>
   <zorder>label_voltaje</zorder>
   <zorder>label_x</zorder>
   <zorder>line_y</zorder>
   <zorder>pushButton_add</zorder>
   <zorder>line_x</zorder>
   <zorder>pushButton_tabla</zorder>
   <zorder>pushButton_mostrar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar"/>
    <addaction name="actionCargar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionCargar">
   <property name="text">
    <string>Cargar</string>
   </property>
  </action>
  <action name="actionLista">
   <property name="text">
    <string>Lista</string>
   </property>
  </action>
  <action name="actionTabla">
   <property name="text">
    <string>Tabla</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
